drafts/draft-lcurley-moq-lite.md

```
@@ -0,0 +1,1185 @@
+- Expires
+- ContentExists
+- Largest Group ID
+- Largest Object ID
+- Parameters
+- Subgroup ID
+- Object ID
+- Object Status
+- Extension Headers
+
+
+# Security Considerations
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)